fluentui_drawer/src/main/java/com/microsoft/fluentui/bottomsheet/BottomSheetItem.kt

```diff
@@ -48,6 +50,7 @@ class BottomSheetItem : Parcelable {
         id: Int = NO_ID,
         @DrawableRes imageId: Int = NO_ID,
         title: String,
+        roleDescription: String = "Button",
```

Comment:
add it as last parameter here & other places.

Reply:
Done